client/blocks/reader-post-card/index.jsx

```diff
@@ -94,6 +231,50 @@ export default class ReaderPostCard extends React.Component {
 		}
 	}
 
+	triggerStreamPixelRequest = ( type, user, key, timeOnScreen ) => {
```

Comment:
should probably add the guts of this to the analytics middleware and just dispatch a tracking action instead of embedding it in the trackable. In fact, we should pass Trackable the action creator to use when firing the action. That way we can reuse trackable for other things.